build.gradle.kts

```diff
@@ -36,6 +38,7 @@ subprojects {
 
   /** Specifies JDK-11 */
   java { toolchain { languageVersion.set(JavaLanguageVersion.of(11)) } }
+  jacoco { toolVersion = "0.8.10" }
```

Comment:
nit: Can we move the versions into [`libs.versions.toml`](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/gradle/libs.versions.toml)?